### PR TITLE
Add flash-messages

### PR DESCRIPTION
**Pull request by matzimowski**
👶  + 🍼  === 🐗 

screen shot 2016-06-07 at 09 57 04

### CODE REVIEW

File: src/components/flash-messages/_flash-messages.scss [hunk 1]
```diff
@@ -0,0 +1,27 @@
+.mint-flash-messages {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  flex-direction: column;
+  position: absolute;
+  left: 0;
+  right: 0;
+  top: 100%;
```
Comment (kdzwinel) [inline src/components/flash-messages/_flash-messages.scss]: IMO positon+left+right+top are specific to symfony/web-client and shouldn't be here. @aju @vergilius WDYT?

Comment (matzimowski) [inline src/components/flash-messages/_flash-messages.scss]: i know it sounds ridiculous but i think you are right.

Comment (matzimowski) [inline src/components/flash-messages/_flash-messages.scss]: you are right to be wrong

Comment (aju) [inline src/components/flash-messages/_flash-messages.scss]: +1

File: src/components/flash-messages/_flash-messages.scss [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    cursor: pointer;
+
+    &--error {
+      background-color: $peachPrimary;
+    }
+
+    &--success {
+      background-color: $mintPrimary;
+    }
+  }
```
Comment (aju) [inline src/components/flash-messages/_flash-messages.scss]: do we have info or warning ?

File: src/components/flash-messages/_flash-messages.scss [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+  right: 0;
+  top: 100%;
+
+  &__message {
+    padding: 0 gutter(1);
+    line-height: rhythm(1);
+    text-align: center;
+    background-color: $bluePrimary;
+    width: 100%;
+    cursor: pointer;
```
Comment (aju) [inline src/components/flash-messages/_flash-messages.scss]: I'm not sure about this - maybe add it as modifier

File: src/components/flash-messages/_flash-messages.scss [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+  justify-content: center;
+  flex-direction: column;
+  position: absolute;
+  left: 0;
+  right: 0;
+  top: 100%;
+
+  &__message {
+    padding: 0 gutter(1);
+    line-height: rhythm(1);
```
Comment (aju) [inline src/components/flash-messages/_flash-messages.scss]: I would like to have here flex and aling items on it

File: src/components/flash-messages/_flash-messages.scss (hunk 1 above)
Comment (matzimowski) [inline src/components/flash-messages/_flash-messages.scss]: ok

File: src/components/flash-messages/_flash-messages.scss (hunk 4 above)
Comment (matzimowski) [inline src/components/flash-messages/_flash-messages.scss]: ok

File: src/components/flash-messages/_flash-messages.scss (hunk 2 above)
Comment (matzimowski) [inline src/components/flash-messages/_flash-messages.scss]: i have added a 'info' modifier

**Comment by matzimowski** [comment]
i have updated name to 'mint-flash'

File: src/components/buttons/_buttons-mixins.scss [hunk 5]
```diff
@@ -43,7 +43,9 @@
 
 @mixin mintButtonPrimaryThreeColorVariant($buttonColor, $gradientColor, $buttonShadowColor: null) {
   background-color: $buttonColor;
+  // scss-lint:disable NameFormat
```
Comment (aju) [inline src/components/buttons/_buttons-mixins.scss]: Waaaaaaat? ??? ? ?? ?? ??? ?? ? ?

**Comment by kdzwinel** [comment]
ok

File: src/components/buttons/_buttons-mixins.scss (hunk 5 above)
Comment (matzimowski) [inline src/components/buttons/_buttons-mixins.scss]: linter wanted me to do this

**Comment by aju** [comment]
OK

**Pull request merged**